js/editor.js

```diff
@@ -162,6 +164,19 @@ class LoadSketch {
             this.online = false;
         }
     }
+
+    async restGet(url){
+        let getRequest = new Request(REST_URL+url+'.json');
+        let xyz;
+
+        return await fetch(getRequest,{method:'GET'})
```

Comment:
This restGet function is the only thing that needs to be left of this class.
 I would rename it to a function called `httpGetSketch`.

Then you can  just do ```await fetch(`${SKETCH_URL}/${id}.json`)```?  Also, don't mix and match `.then` and await. Just do 
```
const res = await fetch(....);
const json = await res.json();
return json;
```

Reply:
Should there be a function inside `LoadSketch` class or should `loadSketch` be a standalone function?